test/Spec.hs

```diff
@@ -322,3 +330,195 @@ integrationTests = testGroup "integrationTests" $
       isUbuntu = ("ubuntu" `T.isPrefixOf`)
   in map testFontset $ zip [0..] $ map T.unpack $ filter (not . isUbuntu)
      $ map fst $ uFontsets uiOptions
+
+itemKindUnitTests :: TestTree
+itemKindUnitTests = testGroup "itemKindUnitTests" $
+  let customRules = RK.RuleContent
+                        { rtitle = ""
+                        , rXmax = 0
+                        , rYmax = 0
+                        , rexeVersion = makeVersion []
+                        , rcfgUIName = ""
+                        , rcfgUIDefault = ("", Ini.emptyConfig)
+                        , rwriteSaveClips = 0
+                        , rleadLevelClips = 0
+                        , rscoresFile = ""
+                        , rnearby = 0
+                        , rstairWordCarried = []
+                        , rsymbolProjectile = '0'
+                        , rsymbolLight      = '0'
+                        , rsymbolTool       = '0'
+                        , rsymbolSpecial    = '0'  -- don't overuse, because it clashes with projectiles
+                        , rsymbolGold       = '0'  -- also gems -- '$'
+                        , rsymbolNecklace   = '*'
+                        , rsymbolRing       = '0'
+                        , rsymbolPotion     = '0'  -- concoction, bottle, jar, vial, canister
+                        , rsymbolFlask      = '0'
+                        , rsymbolScroll     = '0'  -- book, note, tablet, remote, chip, card
+                        , rsymbolTorsoArmor = '0'
+                        , rsymbolMiscArmor  = '0'
+                        , rsymbolClothes    = '0'
+                        , rsymbolShield     = '0'
+                        , rsymbolPolearm    = '0'
+                        , rsymbolEdged      = '0'
+                        , rsymbolHafted     = '0'
+                        , rsymbolWand       = '0'  
+                        , rsymbolFood       = ','  -- also body part; distinct from floor: not middle dot
+                        }
+  in 
+  [ testCase "overlonginame_validateSingle_errs" $
+      validateSingle Content.RuleKind.standardRules ItemKind
```

Comment:
You may find it useful to define one `ItemKind` in the `let` above and then just vary it below, to focus reader's attention to the changed details. I'd definitely find it easier to understand.